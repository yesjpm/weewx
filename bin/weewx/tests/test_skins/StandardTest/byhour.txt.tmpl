#errorCatcher Echo
                   TEST HOURS ITERABLE
---------------------------------------------------------------------------------------
#for $_hour in $day.hours
$_hour.end $_hour.outTemp.max
#end for
---------------------------------------------------------------------------------------
    
                   TEST USING ALTERNATE BINDING
---------------------------------------------------------------------------------------
#for $_hour in $day.hours
$_hour($data_binding='alt_binding').end $_hour($data_binding='alt_binding').outTemp.max
#end for
---------------------------------------------------------------------------------------

