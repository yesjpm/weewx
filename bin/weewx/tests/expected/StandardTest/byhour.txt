                   TEST HOURS ITERABLE
---------------------------------------------------------------------------------------
01:00 46.8F
02:00 43.1F
03:00 40.5F
04:00 39.1F
05:00 39.6F
06:00 41.6F
07:00 44.7F
08:00 48.8F
09:00 53.6F
10:00 58.8F
11:00 63.9F
12:00    N/A
13:00    N/A
14:00    N/A
15:00    N/A
16:00    N/A
17:00    N/A
18:00    N/A
19:00    N/A
20:00    N/A
21:00    N/A
22:00    N/A
23:00    N/A
00:00    N/A
---------------------------------------------------------------------------------------
    
                   TEST USING ALTERNATE BINDING
---------------------------------------------------------------------------------------
01:00 12.1F
02:00 10.8F
03:00 10.1F
04:00 10.4F
05:00 11.4F
06:00 13.0F
07:00 15.1F
08:00 17.5F
09:00 20.1F
10:00 22.6F
11:00 25.1F
12:00    N/A
13:00    N/A
14:00    N/A
15:00    N/A
16:00    N/A
17:00    N/A
18:00    N/A
19:00    N/A
20:00    N/A
21:00    N/A
22:00    N/A
23:00    N/A
00:00    N/A
---------------------------------------------------------------------------------------

